test/unit/parsers/AccessibilityStatementYamlParserSpec.scala

```diff
@@ -412,18 +410,16 @@ class AccessibilityStatementYamlParserSpec extends AnyWordSpec with Matchers wit
       val malformed = """- 1
             |2""".stripMargin
       val parsed    = parser.parse(malformed)
-      parsed.left.value.getMessage should startWith(
-        "while scanning a simple key"
-      )
+
+      parsed should matchPattern { case Left(_) => }
```

Comment:
No longer checked for message, only that it is a `Left`, is that intentional?

Reply:
Hi Jo, Yes there was a comment from Dave as to why we should just test for Left instead of testing for the content as Left as that would be testing Circe itself, we just care that we are getting a left.